backend/collab_service/README.md

```diff
@@ -0,0 +1,29 @@
+# PeerPrep Backend
+
+## Project Overview
+The PeerPrep backend provides a service for managing and querying questions used for technical interview practice. It supports creating, retrieving, and deleting questions.
```

Comment:
Update this line to what the collab service does

Reply:
README.md updated, thanks !